servers/braze-content-proxy/src/routes/digest.ts

```diff
@@ -40,17 +40,20 @@ function transformToBrazePayload(
     return [];
   }
   const savedItems: BrazeSavedItem[] = edges
-    .filter((edge) => edge.node.item.__typename === 'Item')
+    .filter((edge) => edge?.node?.item.__typename === 'Item')
     .map((edge) => {
-      const item = edge.node.item;
-      if (item.__typename === 'Item') {
+      const item = edge?.node?.item;
+      if (item != null && item.__typename === 'Item') {
         return {
-          title: item.preview.title,
-          imageUrl: item.preview.image?.cachedImages[0].url,
-          url: item.preview.url,
+          title: item.preview?.title ?? null,
+          imageUrl: item.preview?.image?.cachedImages?.at(0)?.url ?? null,
+          url: item.preview?.url ?? null,
         };
       }
-    });
+      return null;
+    })
+    // Added due to strict null checks and types above not inferring
+    .filter((item) => item != null) as BrazeSavedItem[];
```

Comment:
it's because your map above can return null

Reply:
@kschelonka found a better way